README.md

```diff
@@ -30,76 +41,90 @@ $ helm install ohmyglb ohmyglb/ohmyglb
 See [values.yaml](https://github.com/AbsaOSS/ohmyglb/blob/master/chart/ohmyglb/values.yaml)
 for customization options.
 
-### Local Playground
-
-####  Deploy local cluster
-
-```sh
-$ make deploy-local-cluster
+### Local Playground Install
+
+#### Environment prerequisites
+
+ - install **GO 1.14**
+ 
+ - install **GIT**     
+    - ensure SSH key is set for your account; see https://github.com/settings/keys  
```

Comment:
Imho his part is not required. People who hangout on github are usually in git context

Also I am using personal access token instead of ssh key for additional security so this recommendation might be misleading. Let's not duplication official git/github docs here